Sources/RichTextKit/Coordinator/RichTextLinkSetter.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  RichTextLinkSetter.swift
+//
+//
+//  Created by Dominik Bucher on 12.12.2023.
+//
+
+import Foundation
+#if macOS
+import AppKit
+#else
+import UIKit
+#endif
+
+public class RichTextLinkSetter {
+#if macOS
+    let textView: NSTextView
```

Comment:
You can use the newly pushed `TextViewRepresentable` to avoid having two initializers.

Reply:
But since we have the `(textView as? RichTextView)?.renderLinks` cast, why not just require a `RichTextView `?